packages/orchestration/src/exos/packet-tools.js

```diff
@@ -44,9 +44,9 @@ const just = obj => {
  */
 
 /**
- * @typedef {Pick<
+ * @typedef {Omit<
  *   import('../cosmos-api').IBCMsgTransferOptions,
- *   'timeoutHeight' | 'timeoutTimestamp'
+ *   'memo' | 'forwardOpts'
  * >} PacketTimeout
  */
```

Comment:
I'm not sure this is correct - more so the existing code. I see it used for `.sendThenWaitForAck(.., opts)`, but that opts bag looks more like vow context and less IBCMsgTransferOpts.